### PR TITLE
Package resides now in ./package

### PR DESCRIPTION
**Pull request by jakob-r**
All necessary shiny app files are linked from inst/application

### CODE REVIEW

**Comment by jakob-r** [comment]
after PR it should be possible to install it via `devtools::install_github("mlr-org/shiny", subdir = "package")`

File: package/DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Title:  A graphical user interface for machine learning in R
+Description:
+  shinyMlr wraps the functionalities of the R-package mlr
+  into a graphical user-interface built with shiny.
+  This enables the user to conduct all steps of the machine learning workflow from his browser.
+Authors@R: c(
+  person("Florian", "Fendt", email = "flo_fendt@gmx.de",
+    role = c("aut", "cre")),
+  person("Stefan", "Coors", email = "stefan.coors@gmx.net",
+    role = "aut"))
```
Comment (florianfendt) [inline package/DESCRIPTION]: feel free to add you here as author :)

File: package/DESCRIPTION [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  R (>= 3.3.2)
+Imports:
+  shiny,
+  shinythemes
+Suggests:
+  mlr,
+  testthat,
+  RSelenium,
+  roxygen2,
+  knitr,
```
Comment (florianfendt) [inline package/DESCRIPTION]: Now the whole app is in the package we should also add mlr-imports as suggests here, right?

File: package/R/runShinyMlr.R [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#' Run a local instance of shinyMlr
+#'
+#' @param ... [\code{any}]\cr
+#'   Additional arguments passed to shiny's
+#'   \code{runApp()} function.
+#' @examples
+#' \dontrun{
+#'   runShinyMlr()
+#' }
+#' @seealso downloadShinyMlr
```
Comment (florianfendt) [inline package/R/runShinyMlr.R]: @seealso needs to be removed

**Comment by florianfendt** [comment]
@jakob-r 
Thx heaps again for the contribution.
If you have the time implement these 2/3 small changes I reviewed and merge your PR.
No worries if not then we'll do this.
Cheers

**Comment by florianfendt** [comment]
@jakob-r 
Damn you're fast :)
Feel free to merge.
Don't delete the branch `package` though please and leave as is, since the UseR application contained the old workflow of the package and I want that to work when they evaluate our application. I'll open an issue for that so we don't forget deleting the branch when our application got approved.

**Comment by jakob-r** [comment]
It's mainly your repository - I would never dare to delete anything. That's your business.

**Pull request merged**